salt/modules/rpm.py

```diff
@@ -128,21 +128,27 @@ def bin_pkg_info(path, saltenv='base'):
     return ret
 
 
-def list_pkgs(*packages):
+def list_pkgs(*packages, **kwargs):
     '''
     List the packages currently installed in a dict::
 
         {'<package_name>': '<version>'}
 
+    root
+        use root as top level directory (default: "/")
+
     CLI Example:
 
     .. code-block:: bash
 
         salt '*' lowpkg.list_pkgs
     '''
     pkgs = {}
-    cmd = ['rpm', '-q' if packages else '-qa',
-           '--queryformat', r'%{NAME} %{VERSION}\n']
+    cmd = ['rpm']
+    if 'root' in kwargs:
+        cmd.extend(['--root', kwargs['root']])
```

Comment:
This is bug. Correctly would be:
```python
if kwargs.get('root'):
   ...
```

Reply:
Oh, you are referring to the line up. I will change. But still I do not see the error. Is there a case where __getitem__ is not defined for kwargs?

Reply:
Ah nono. I get it now, the goal is to make sure that is not None , '' or 0. But this pattern can exclude 0, and sometimes is a valid value.